plugins/mobile-plugin/src/test/java/pagefactory/mobile/pages/building/BuildingPage.java

```diff
@@ -0,0 +1,33 @@
+package pagefactory.mobile.pages.building;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import ru.sbtqa.tag.pagefactory.mobile.utils.SwipeUtils;
+import ru.sbtqa.tag.pagefactory.mobile.MobilePage;
+import ru.sbtqa.tag.pagefactory.annotations.ActionTitle;
+import ru.sbtqa.tag.pagefactory.annotations.ElementTitle;
+import ru.sbtqa.tag.pagefactory.annotations.PageEntry;
+import ru.sbtqa.tag.pagefactory.utils.ExpectedConditionsUtils;
+
+@PageEntry(title = "Здание")
+public class BuildingPage extends MobilePage {
+
+    @ElementTitle(value = "Заголовок")
+    @FindBy(xpath = "//android.widget.TextView[@text='Здание']")
+    protected WebElement lbl;
+
+    @FindBy(id = "ru.sberbank.sbt.eso:id/suggest_list")
+    protected WebElement list;
+
+    public BuildingPage(WebDriver driver) {
+        super(driver);
+        ExpectedConditionsUtils.waitUntilElementPresent(lbl);
+        ExpectedConditionsUtils.waitUntilElementPresent(list);
+    }
+
+    @ActionTitle("выбирает случайное значение из списка")
+    public void getRandomItem() {
+//	    clickWebElement(Utils.getRandomItem("ru.sberbank.sbt.eso:id/suggest_item_value"));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)